From 9a690622915308d0832e5565b4f0ce66563fa09f Mon Sep 17 00:00:00 2001
From: Juan Quintela <quintela@redhat.com>
Date: Tue, 14 Jan 2014 15:07:14 +0100
Subject: [PATCH 03/40] memory: cpu_physical_memory_set_dirty_range() return void

RH-Author: Juan Quintela <quintela@redhat.com>
Message-id: <1389712071-23303-4-git-send-email-quintela@redhat.com>
Patchwork-id: 56659
O-Subject: [RHEL7 qemu-kvm PATCH 03/40] memory: cpu_physical_memory_set_dirty_range() return void
Bugzilla: 997559
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

Signed-off-by: Juan Quintela <quintela@redhat.com>
Reviewed-by: Orit Wasserman <owasserm@redhat.com>
Reviewed-by: Eric Blake <eblake@redhat.com>
(cherry picked from commit 78d0042642a415b56b3c53a6044f30eff47bf69c)
Signed-off-by: Juan Quintela <quintela@trasno.org>
---
 memory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 memory.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/memory.c b/memory.c
index 75ca281..fc11859 100644
--- a/memory.c
+++ b/memory.c
@@ -1078,7 +1078,7 @@ void memory_region_set_dirty(MemoryRegion *mr, hwaddr addr,
                              hwaddr size)
 {
     assert(mr->terminates);
-    return cpu_physical_memory_set_dirty_range(mr->ram_addr + addr, size, -1);
+    cpu_physical_memory_set_dirty_range(mr->ram_addr + addr, size, -1);
 }
 
 bool memory_region_test_and_clear_dirty(MemoryRegion *mr, hwaddr addr,
-- 
1.7.1

